gallery_dl/extractor/urlgalleries.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 as
+# published by the Free Software Foundation.
+
+"""Extractors for https://urlgalleries.net/"""
+
+from .common import GalleryExtractor
+from .. import text
+
+
+class UrlgalleriesGalleryExtractor(GalleryExtractor):
+    """Base class for Urlgalleries extractors"""
+    category = "urlgalleries"
+    root = "urlgalleries.net"
+    directory_fmt = ("{category}", "{title}")
+    pattern = r"(?:https?://)([^/?#]+)?\.urlgalleries\.net/([^/?#]+)/([^/?#]+)"
```

Comment:
```suggestion
    pattern = r"(?:https?://)([^/?#]+\.)?urlgalleries\.net/([^/?#]+)/([^/?#]+)"
```
otherwise it would match `https://.urlgalleries.net`